google/search/tool.gpt

```diff
@@ -28,4 +28,4 @@ Google Search
 
 ---
 !metadata:*:icon
-https://cdn.jsdelivr.net/npm/@phosphor-icons/core@2/assets/duotone/list-magnifying-glass-duotone.svg
+/admin/assets/google_icon_small.png
```

Comment:
looking at the paths in main, it seems like this should be `/admin/public/assets/google_icon_small`?